python/src/tests/tasks/test_list_genes.py

```diff
@@ -76,58 +73,6 @@ def test_construct_request_cleans_regex(self):
         request = ListGenes(self.clean_regex)._construct_request()
         assert request["geneNamesFilter"] == "LIN"
 
-    def test_construct_request_cleans_regex(self):
+    def test_construct_request_preserves_begin_and_end_with(self):
         request = ListGenes(self.partial_clean_regex)._construct_request()
         assert request["geneNamesFilter"] == "^$LIN"
-
-"""
-    def test_descending(self):
```

Comment:
why are these tests deleted?

Reply:
because they've been migrated to R, but then forgotten to be deleted after the migration

Reply:
they were just commented out